runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,16 @@
+# https://quarkus.io/version/main/guides/extension-metadata#quarkus-extension-yaml
+name: "Logging Splunk"
+description: "Send logs to a Splunk HTTP Event Collector (HEC)"
+sponsor: "Amadeus IT Group"
```

Comment:
@wherka-ama Could you approve this and confirm whether we should use "Amadeus" vs  "Amadeus IT Group" ?

Reply:
@rquinio1A : It's okay as is. Thanks again for your great work guys!